tests/bears/BearTest.py

```diff
@@ -1,10 +1,14 @@
 import multiprocessing
+import shutil
 import unittest
+
 from os.path import abspath, exists, isfile, join, getmtime
-import requests_mock
-import shutil
 from time import sleep
 
+import requests
+
+import requests_mock
```

Comment:
Should there be a space b/w the imports (not aware of the import styles of coala though :sweat_smile: , I guess they're not documented anywhere) But according to PEP8
> Imports should be grouped in the following order:
    standard library imports
    related third party imports
    local application/library specific imports